emBODY/eBcode/arch-dspic/board/2foc/appl/2FOC-V3/src/ecan.c

```diff
@@ -6,7 +6,7 @@
 
 #include <p33FJ128MC802.h>
 #include <string.h>
-
+#include "stdint.h"
```

Comment:
Thanks @giovannipizzolante , I have a nitpick (i.e. a comment that you can safely ignore, see https://google.github.io/eng-practices/review/reviewer/looking-for.html#style). 

In C/C++, the main difference between includes that are added with `<include-name.h>` vs the one that are included with  `"include-name.h"` is that the latter one, i.e. the one with the quote, also looks in the same directory of this file to search for the included .h file (see https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename). For this reason, the style with which normally system includes are added is the one with the `<` , `>`. 
```suggestion
#include <stdint.h>
```

Reply:
thanks @traversaro !